### ISSUE TITLE
Error when authenticating with private key

### ISSUE THREAD

**Issue by mattpfeffer** [google/earthengine-api#92]
As of v0.1.180 I'm receiving a '_ReferenceError: document is not defined_' when calling `ee.data.authenticateViaPrivateKey()` on the Javascript API. I can confirm the key is correct. Rolling back to the previous release resolves the problem.

**Comment by cgmorton** [comment]
I'm not sure if it is exactly related, but I am having a similar problem on the Python side with version 0.1.180.  

While trying to figure out the problem I created a new key file and noticed that the "token_uri" line changed.  I can get EE to initialize (with v0.1.180) with my old key if I change the following line in the JSON key file from:
```
  "token_uri": "https://accounts.google.com/o/oauth2/auth",
```
to:
```
  "token_uri": "https://oauth2.googleapis.com/token",
```

**Comment by donmccurdy** [comment]
@cgmorton I believe that's a different issue, could you open a new bug? Thanks!

@mattpfeffer thanks, this should be fixed on the next release.

**Comment by donmccurdy** [comment]
The original issue is fixed with 0.1.184.
